src/cf-base.less

```diff
@@ -417,16 +435,24 @@ a {
 p,
 li,
 dd {
-    // Restrict bottom borders to inline text links ...
-
+    // Restrict underlines to inline text links ...
     a {
-        border-bottom-width: 1px;
+        text-decoration: underline;
+
+        .cssgradients & {
+            background-size: (2px + 2px) 2px;
```

Comment:
Is there somewhere that explains the significance of `(2px + 2px)`? I'm assuming one is for the width of the underline and maybe the other is an offset from the text?